assets/javascripts/stuff-to-do.js

```diff
@@ -222,17 +222,26 @@ jQuery(function($) {
   };
 
   var bindContextMenuToTimeGrid = function() {
-    $("#time-grid-table tr td.time-grid-date").
-      contextMenu(
+
+    $.contextMenu(
         {
-          menu: 'time-grid-menu',
-          menuCssName: 'context-menu'
-        },
-        function(action, el, pos) {
-          timeLogFacebox(parseIssueId(el.parent()),
-          parseDateFromGrid(el));
+		  selector:"#time-grid-table tr td.time-grid-date",
+		  callback:function(key,options) {
+				  var el =  options.$trigger;
+				timeLogFacebox(parseIssueId(el.parent()),
+				parseDateFromGrid(el));
+				},
+		  items:{firstCommand: {name: $("#log-time").text() }}
         }
+        
       );
+	  
+	  $("#time-grid-table tr td.time-grid-date").hover(function(){
+			$(this).css("background-color","yellow");
+		},function(){
+			$(this).css("background-color","white");
+		});
+
```

Comment:
I'm getting the following client-side error when using this:

```console
stuff-to-do.js?1633982868:200 Uncaught TypeError: Cannot read properties of undefined (reading 'split')
    at parseIssueId (stuff-to-do.js?1633982868:200)
    at s.fn.init.callback (stuff-to-do.js?1633982868:231)
    at HTMLLIElement.itemClick (jquery.contextMenu.js?1633982868:894)
    at HTMLDocument.dispatch (jquery-3.5.1.min.js?1633982868:2)
    at HTMLDocument.v.handle (jquery-3.5.1.min.js?1633982868:2)
```

Is there a conflict between the jquery.contextMenu plugin and Redmine 4.2.1's own custom contextMenu handler